### PR TITLE
Fixed all linting issues

### PR DESCRIPTION
**Pull request by kdsinghcoder**
Fixed issue: #70

### CODE REVIEW

File: src/account/account.service.spec.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { getModelToken, MongooseModule } from '@nestjs/mongoose';
+// import { getModelToken, MongooseModule } from '@nestjs/mongoose';
 import { Test, TestingModule } from '@nestjs/testing';
-import { Model } from 'mongoose';
+// import { Model } from 'mongoose';
 import { ApplicationService } from '../application/application.service';
 import { UserService } from '../user/user.service';
-import { User, UserDocument, UserSchema } from '../user/user.schema';
+// import { User, UserDocument, UserSchema } from '../user/user.schema';
 import { AccountService } from './account.service';
```
Comment (YashKumarVerma) [inline src/account/account.service.spec.ts:7]: remove the lines, no need to keep as comments.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Comment by YashKumarVerma** [comment]
also, fill the PR body & title properly.

File: src/account/account.service.spec.ts (hunk 1 above)
Comment (kdsinghcoder) [reply src/account/account.service.spec.ts]: Done

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Pull request closed without merging**